src/bin/tikv-ctl.rs

```diff
@@ -67,15 +68,22 @@ fn new_debug_executor(
 ) -> Box<DebugExecutor> {
     match (host, db) {
         (None, Some(kv_path)) => {
-            let db = util::rocksdb::open(kv_path, ALL_CFS).unwrap();
-            let raft_db = if let Some(raft_path) = raft_db {
-                util::rocksdb::open(raft_path, &[CF_DEFAULT]).unwrap()
-            } else {
-                let raft_path = PathBuf::from(kv_path).join("../raft");
-                util::rocksdb::open(raft_path.to_str().unwrap(), &[CF_DEFAULT]).unwrap()
-            };
-            Box::new(Debugger::new(Engines::new(Arc::new(db), Arc::new(raft_db)))) as
-                Box<DebugExecutor>
+            let cfg = TiKvConfig::default();
```

Comment:
can we need to support a flag to pass the tikv config instead? If no config, we can use the default config.

Reply:
I think the default configuration is enough.

Reply:
It would be better to read the configuration file from the tikv that it is trying to open, in case some users use some special configurations.